schema/proto3/subnet.proto

```diff
@@ -27,36 +27,27 @@ message SubnetConfiguration {
     uint32 format_version = 1;
     uint32 revision_number = 2;
 
-    string id = 3;
-    string project_id = 4;
-    string vpc_id = 5;
-    string name = 6;
-    string cidr = 7;
-    int64 tunnel_id = 8;
+    string request_id = 3;
+    string id = 4;
+    NetworkType network_type = 5;
```

Comment:
Should we move network_type from port directly to network, instead of subnet? Do you plan to support two subnets in the same network with different values of network_type?

Reply:
putting it in subnet level will allow us to support:
1) two subnets in the same network with different network_type
2) each network with n subnet will support one network_type only

@cj-chung can you help to find out what openstack neutron supports?

Note that we have the notion to look up subnet info, but we don't have the notion to look up vpc info (yet).

Reply:
Neutron by default supports one network, one network type, and one segmentation id. 

Ref: https://docs.openstack.org/api-ref/network/v2/?expanded=create-network-detail#create-network

Regarding two subnets with different network_type, DP will be pretty challenging.

Reply:
Also, it supports one network with multiple segments, each with different network type and segmentation id. 

```json
    "network": {
        "segments": [
            {
                "provider:segmentation_id": 2,
                "provider:physical_network": "public",
                "provider:network_type": "vlan"
            },
            {
                "provider:physical_network": "default",
                "provider:network_type": "flat"
            }
        ],
        "name": "net1",
        "admin_state_up": true,
        "qos_policy_id": "6a8454ade84346f59e8d40665f878b2e"
    }
} 
```